tests/v2/validation/Jenkinsfile.recurring

```diff
@@ -0,0 +1,310 @@
+#!groovy
+node {
+    def rootPath = "/root/go/src/github.com/rancher/rancher/"
+    def workPath = "/root/go/src/github.com/rancher/rancher/tests/v2/validation/"
+    def jobName = "${JOB_NAME}"
+    if (jobName.contains('/')) { 
+      jobNames = jobName.split('/')
+      jobName = jobNames[jobNames.size() - 1] 
+    }
+    def golangTestContainer = "${jobName}${env.BUILD_NUMBER}-golangtest"
+    def buildTestContainer = "${jobName}${env.BUILD_NUMBER}-buildtest"
+    def configGeneratorContainer = "${jobName}${env.BUILD_NUMBER}_generator"
+    def cleanupTestContainer = "${jobName}${env.BUILD_NUMBER}-cleanuptest"
+    def imageName = "rancher-recurring-runs-validation-${jobName}${env.BUILD_NUMBER}"
+    def validationVolume = "RecurringRunsSharedVolume-${jobName}${env.BUILD_NUMBER}"
+    def testsDir = "/root/go/src/github.com/rancher/rancher/tests/v2/validation/"
+    def configsDir = "cattle-configs"
+    def patchedConfigsDir = "patched-cattle-configs"
+    def testResultsOut = "results.xml"
+    def envFile = ".env"
+    def rancherConfig = "rancher_env.config"
+    def branch = "release/v2.7"
```

Comment:
this should default to 2.8 in this forwardport

Reply:
Agreed

Reply:
fixed it